src/libs/NextStepUtils.ts

```diff
@@ -40,11 +40,13 @@ function parseMessage(messages: Message[] | undefined) {
 
     messages?.forEach((part) => {
         const isEmail = Str.isValidEmail(part.text);
-        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
         let tagType = part.type ?? 'span';
         let content = Str.safeEscape(part.text);
 
-        if (isEmail) {
+        if (currentUserEmail === part.text) {
```

Comment:
hmm I didn't test that, I think we should still show `You` if you you have a display name.

Reply:
The `messages` contain this:

```
[
    {
        "text": "Waiting for "
    },
    {
        "text": "Approver's Name Approver's LastName",
        "type": "strong",
        "tooltipText": "approver@wp-advanced-approval4.com (click to copy)",
        "clickToCopyText": "approver@wp-advanced-approval4.com"
    },
    {
        "text": " to "
    },
    {
        "text": "pay5"
    },
    {
        "text": " %expenses."
    }
]
```

I'm going to use the `part.clickToCopyText` as a fallback to check because the email should be there if it is not in `part.text`. I don't expect to find the current user email in cases other than when we are trying to show the current user email (there shouldn't be false positives).